.gitignore

```diff
@@ -5,6 +5,7 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 lerna-debug.log*
+.idea
```

Comment:
GitHub has a repository with good gitignore templates. They have a [`.gitignore` template for Jetbrains IDE's](https://github.com/github/gitignore/blob/main/Global/JetBrains.gitignore).

Maybe we should copy/paste some things from them? Before copying, check if their license is okay for us: [`github/gitignore` license file](https://github.com/github/gitignore/blob/main/LICENSE).